blazar/manager/service.py

```diff
@@ -437,6 +446,26 @@ def update_lease(self, lease_id, values):
                     'Please enter valid reservation IDs. Invalid reservation '
                     'IDs are: %s' % ','.join([str(id) for id in invalid_ids]))
 
+            try:
+                [
+                    self.plugins[r['resource_type']] for r
```

Comment:
What is this doing, exactly? It seems like it's checking if the user is providing a weird/unsupported resource_type, in which case `UnsupportedResourceType` is more appropriate to raise here.